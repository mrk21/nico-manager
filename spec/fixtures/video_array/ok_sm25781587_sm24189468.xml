<?xml version="1.0" encoding="UTF-8"?>
<nicovideo_video_response status="ok">
  <count>2</count>
  <video_info>
    <video>
      <id>sm25781587</id>
      <user_id>12170860</user_id>
      <deleted>3</deleted>
      <title>【手描き】フィンランド人類は繁栄しました</title>
      <description>トーベ・ヤンソンさん生誕100周年（2014）おめでとうございます。ムーミンさん生誕70周年（2015）おめでとうございます。ということで描いてみました。（参考動画）　インド人類は繁栄しました　sm18393920　【第10回MMD杯本選】MMDは繁栄しました　sm20124530&lt;br&gt;&lt;br&gt;&lt;span style="background:#FFCCCC; color:#FF0000;"&gt;&lt;span id="deleted_message_default"&gt;この動画はMOOMIN CHARACTERS OY LTDの申立により、著作権侵害として削除されました。&lt;br&gt;対象物: MOOMIN CHARACTERS OY LTDが著作権を有する著作物&lt;/span&gt;&lt;span id="deleted_message_ext" value="This video has been deleted due to the MOOMIN CHARACTERS OY LTD's notice of motion."&gt;&lt;/span&gt;&lt;/span&gt;</description>
      <length_in_seconds>119</length_in_seconds>
      <size_low>6449212</size_low>
      <movie_type>mp4</movie_type>
      <thumbnail_url>http://tn-skr4.smilevideo.jp/smile?i=25781587</thumbnail_url>
      <upload_time/>
      <first_retrieve>2015-03-14T00:15:00+09:00</first_retrieve>
      <default_thread>1426259701</default_thread>
      <view_counter>409044</view_counter>
      <mylist_counter>29810</mylist_counter>
      <option_flag_ichiba>0</option_flag_ichiba>
      <option_flag_community>0</option_flag_community>
      <option_flag_domestic>0</option_flag_domestic>
      <option_flag_comment_type>0</option_flag_comment_type>
      <option_flag_adult>0</option_flag_adult>
      <option_flag_mobile>0</option_flag_mobile>
      <option_flag_economy_mp4>1</option_flag_economy_mp4>
      <option_flag_middle_video>0</option_flag_middle_video>
      <option_flag_mobile_ng_apple>0</option_flag_mobile_ng_apple>
      <main_category>描いてみた</main_category>
      <main_category_key>draw</main_category_key>
    </video>
    <thread>
      <id>1426259701</id>
      <public>1</public>
      <num_res>6572</num_res>
      <community_id/>
    </thread>
    <tags>
      <tag_info>
        <tag>描いてみた</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>楽しいムーミン一家</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>リアルワールド</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>著作権侵害動画</tag>
        <area>jp</area>
      </tag_info>
    </tags>
  </video_info>
  <video_info>
    <video>
      <id>sm24189468</id>
      <user_id>12009682</user_id>
      <deleted>0</deleted>
      <title>自主制作アニメ【鬼斬娘】 ᵗˢᵘᵇᵘʳᵃ</title>
      <description>頭頂部に誇りを生やす化物を「鬼」と呼ぶ世界の、鬼と娘のお話。&lt;br /&gt;&lt;br /&gt;5年前に自主制作した初めての映像作品「鬼斬少女」の続編です。&lt;br /&gt;映像前半は2009年制作、後半は2014年制作です。&lt;br /&gt;5年分の比較も楽しもうと、意図して似たような展開や構図を盛り込みました。&lt;br /&gt;単体はYouTubeでご覧頂けます。&lt;br /&gt;「鬼斬少女(2009)」：https://www.youtube.com/watch?v=fKW0whvVUYQ&lt;br /&gt;「鬼斬娘(2014)」:https://www.youtube.com/watch?v=tfr5iaEE9bQ&lt;br /&gt;&lt;br /&gt;制作　つぶら&lt;br /&gt;[HP]http://www.amatsubu-rabbit.com/</description>
      <length_in_seconds>131</length_in_seconds>
      <size_low>6946811</size_low>
      <movie_type>mp4</movie_type>
      <thumbnail_url>http://tn-skr1.smilevideo.jp/smile?i=24189468</thumbnail_url>
      <upload_time/>
      <first_retrieve>2014-08-08T20:42:05+09:00</first_retrieve>
      <default_thread>1407498125</default_thread>
      <view_counter>506009</view_counter>
      <mylist_counter>28681</mylist_counter>
      <option_flag_ichiba>0</option_flag_ichiba>
      <option_flag_community>0</option_flag_community>
      <option_flag_domestic>0</option_flag_domestic>
      <option_flag_comment_type>0</option_flag_comment_type>
      <option_flag_adult>0</option_flag_adult>
      <option_flag_mobile>0</option_flag_mobile>
      <option_flag_economy_mp4>1</option_flag_economy_mp4>
      <option_flag_middle_video>0</option_flag_middle_video>
      <option_flag_mobile_ng_apple>0</option_flag_mobile_ng_apple>
      <main_category>アニメ</main_category>
      <main_category_key>anime</main_category_key>
    </video>
    <thread>
      <id>1407498125</id>
      <public>1</public>
      <num_res>6083</num_res>
      <community_id/>
    </thread>
    <tags>
      <tag_info>
        <tag>アニメ</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>自主制作アニメ</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>purikoko</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>Rabbit_MACHINE</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>最後まで視聴推奨</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>5年後の月日（進化⁉）</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>ワープ進化</tag>
        <area>jp</area>
      </tag_info>
      <tag_info>
        <tag>作画解放</tag>
        <area>jp</area>
      </tag_info>
    </tags>
  </video_info>
</nicovideo_video_response>
