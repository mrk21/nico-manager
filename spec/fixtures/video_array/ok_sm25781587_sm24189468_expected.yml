---
sm25781587:
  description: 'トーベ・ヤンソンさん生誕100周年（2014）おめでとうございます。ムーミンさん生誕70周年（2015）おめでとうございます。ということで描いてみました。（参考動画）　インド人類は繁栄しました　sm18393920　【第10回MMD杯本選】MMDは繁栄しました　sm20124530<br><br><span
    style="background:#FFCCCC; color:#FF0000;"><span id="deleted_message_default">この動画はMOOMIN
    CHARACTERS OY LTDの申立により、著作権侵害として削除されました。<br>対象物: MOOMIN CHARACTERS OY LTDが著作権を有する著作物</span><span
    id="deleted_message_ext" value="This video has been deleted due to the MOOMIN
    CHARACTERS OY LTD''s notice of motion."></span></span>'
  tags:
    - "描いてみた"
    - "楽しいムーミン一家"
    - "リアルワールド"
    - "著作権侵害動画"
  
sm24189468:
  description: "頭頂部に誇りを生やす化物を「鬼」と呼ぶ世界の、鬼と娘のお話。<br /><br />5年前に自主制作した初めての映像作品「鬼斬少女」の続編です。<br
    />映像前半は2009年制作、後半は2014年制作です。<br />5年分の比較も楽しもうと、意図して似たような展開や構図を盛り込みました。<br
    />単体はYouTubeでご覧頂けます。<br />「鬼斬少女(2009)」：https://www.youtube.com/watch?v=fKW0whvVUYQ<br
    />「鬼斬娘(2014)」:https://www.youtube.com/watch?v=tfr5iaEE9bQ<br /><br />制作　つぶら<br
    />[HP]http://www.amatsubu-rabbit.com/"
  tags:
    - "アニメ"
    - "自主制作アニメ"
    - purikoko
    - Rabbit_MACHINE
    - "最後まで視聴推奨"
    - 5年後の月日（進化⁉）
    - "ワープ進化"
    - "作画解放"
